travis/unit_tests/.gitignore

```diff
@@ -1,4 +1,3 @@
-Makefile
-*_gen.f90
+# Ignore FRUIT files in this directory
+fruit_generator.rb
 fruit_driver.exe
-*.xml
```

Comment:
I think the `xml` file ends up in the main directory. Also are `*_gen.f90` and `Makefile` in here?